.github/workflows/pull_request.yml

```diff
@@ -11,6 +11,50 @@ jobs:
     runs-on: ubuntu-latest
     name: Maven Build
     steps:
+      - name: Free disk space
+        run: |
+          df -h
+          sudo apt-get remove aria2 ansible azure-cli shellcheck rpm xorriso zsync \
```

Comment:
I'm glad to hear that freeing some space helped. I am a bit concerned about maintainability of this command: these packages' names will change over time.

Reply:
I see, thanks for that. Then I can probably reduce the scope of cleaning up and clean up only some cashed docker images on the node.
And what would be if the package is not found? I mean is that will interrup entire build in that case?

Reply:
No, it would provide just a partial cleanup; some packages would be wiped out, some wouldn't.

Reply:
I would suggest using a minimum container (which wouldn't have these dependencies in the first place), but it doesn't seem GitHub actions support that without using a self-hosted runner.

Reply:
We already got the same in VRP.
https://github.com/kiegroup/optaweb-vehicle-routing/pull/385#discussion_r512855544
Lets have this for now?